style.css

```
@@ -1 +1,246 @@
+  }
+  
+.topnav #myLinks {
+    display: none;
+}
+  
+.topnav a {
+    float: left;
+    color: black;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+  
+.topnav a.icon {
+    float: right;
+}
+  
+.topnav a:hover {
+    background-color: rgb(255, 149, 193);
+    color: white;
+    border:rgb(255, 149, 193);
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+}
+
+header {
+    background-color: rgb(33, 248, 248);
+    padding-bottom: 40px;
+    display: flex;
+}
+
+h1 {
+    align-self: flex-end;
+    text-align: right;
+    flex-grow: 2;
+    margin-right: 50px;
+    font-family: 'Lobster', cursive;
+    font-size: 14px;
+}
+
+.logo {
+    width: 150px;
+    height: 150px;
+}
+
+.intro {
+    background-color: rgb(243, 236, 141);
+    text-align: justify;
+    font-size: 14px;
+}
+
+h2, p {
+    font-family: 'Lobster', cursive;
+    margin-left: 40px;
+    padding: 40px 40px 40px 10px;
+}
+
+p {
+    padding-top: 20px;
+}
+
+h2 {
+    padding-bottom: 10px;
+}
+
+.giantCupcake {
+    width: 200px;
+    height: 250px;
+    border: 2px solid rgb(243, 236, 141);
+}
+
+.cakes {
+    width: 100px;
+    height: 75px;
+    margin: 2px;
+    border: 2px solid rgb(255, 149, 193);
+}
+
+.giantCake, .cupcakes {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: rgb(255, 149, 193);
+    margin-top: 20px;
+    padding-top: 40px;
+    padding-bottom: 40px;
+}
+
+.cupcakes {
+    padding: 40px;
+    flex-flow: row wrap;
+    background-color:rgb(243, 236, 141);
+}
+
+footer {
+    background-color: rgb(33, 248, 248);
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+.socialMedia {
+    display: flex;
+    flex-flow: column;
+    width: 20px;
+    height: 20px;
+    padding: 5px;
+    padding-left: 12px;
+    filter: sepia(200%) hue-rotate(280deg) saturate(250%);
+}
+
+.contact {
+    font-size: 12px;
+    text-align: center;
+}
+
+@media (min-width: 420px) {
+    .logo {
+        width: 200px;
+        height: auto;
+    }
+    h1 {
+        font-size: 25px;
+    }
+    .giantCupcake {
+        width: 350px;
+        height: 450px;
+    }
+    .cakes {
+        width: 150px;
+        height: 100px;
+    }
+    .intro, p {
+        font-size: 20px;
+    }
+    .socialMedia {
+        width: 40px;
+        height: 40px;
+        padding-left: 20px;
+        padding-bottom: 10px;
+    }
+    .contact {
+        font-size: 20px;
+    }
+
+@media (min-width: 532px) {
+    .logo {
+        width: 200px;
+        height: auto;
+    }
+    h1 {
+        font-size: 25px;
+    }
+    .giantCupcake {
+        width: 350px;
+        height: 450px;
+    }
+    .cakes {
+        width: 210px;
+        height: 140px;
+    }
+    .intro, p {
+        font-size: 20px;
+    }
+    .socialMedia {
+        width: 40px;
+        height: 40px;
+        padding-left: 20px;
+        padding-bottom: 10px;
+    }
+    .contact {
+        font-size: 20px;
+    }
+
```

```suggestion
}
```
you missed a closing bracket for @media query